app/components/selected_references_component.rb

```diff
@@ -12,7 +12,7 @@ def initialize(application_form, editable: true, show_incomplete: false, is_erro
   end
 
   def show_incomplete_banner?
-    !application_form.references_completed? && show_incomplete
+    !application_form.references_completed? && show_incomplete && editable
```

Comment:
👍🏼 

Will revisit next week, I think we need to move the incomplete banner outside of this component. Makes the logic and tests too difficult to follow.